### DISCUSSION TITLE
Simple graphs and metrics for scores and evaluations

### DISCUSSION THREAD

**Discussion by justinwiley** [langfuse/langfuse#2534] [Ideas]
### Describe the feature or potential improvement

For background, we build AI systems on behalf of customers and use Langfuse to track feedback.  Generally this involves building a RAG based question answering system.  As they interact with the system and leave feedback, they want to see how scores improve or do not improve over time.

The existing score dashboards are a little limited, in terms of seeing a longer time span, and simple metrics around average scores, relative improvements etc.  In addition, evaluations and comments are difficult to see over time.

In general, our customer sponsors are not data scientists or AI engineers, so the nuances of traces and observations are less useful.  Having easily comprehensible charts would be useful for proving the value of the systems we are developing, and give our customers a more concrete understanding of what we are collecting.

Stretch goal: a user role to access just this section - so we could provide customers with their own logins to view graphs

Stretch goal: clustering user comments - providing some way to visualize user free text feedback

**Comment by marcklingen** [comment]
Thanks for raising this, can you share a list of specific charts that would be useful here?
